static/js/user_status.js

```diff
@@ -54,8 +60,11 @@ exports.set_status_text = function (opts) {
 
 exports.initialize = function () {
     _.each(page_params.user_status, function (dct, user_id) {
+        // next line is defensive
```

Comment:
Do we need this?  `page_params.user_status` is a thing with a well-defined integer type; I'd prefer to avoid doing this kind of parsing where it isn't needed because ultimately that'll make moving to typescript (where we might define an expected type for `page_params` and then use that everywhere) harder to read.

Reply:
JSON annoyingly converts keys to strings:

~~~
>>> import json
>>> json.dumps({5: 'five'})
'{"5": "five"}'
~~~

I am about to push a small change that should be more type-friendly.

~~~
-    _.each(page_params.user_status, function (dct, user_id) {
-        // next line is defensive
-        user_id = parseInt(user_id, 10);
+    _.each(page_params.user_status, function (dct, str_user_id) {
+        // JSON does not allow integer keys, so we
+        // convert them here.
+        user_id = parseInt(str_user_id, 10);
~~~

gonna just squash it in

Reply:
missing `const` above, but will be fixed in next push